bindings/java/src/org/sleuthkit/datamodel/SleuthkitCase.java

```diff
@@ -1571,6 +1627,54 @@ public AddImageProcess makeAddImageProcess(String timeZone, boolean addUnallocSp
 	}
 
 	/**
+	 * Gets the the datasource obj ids for the given device_id
+	 *
+	 * @param deviceId device_id
+	 *
+	 * @return A list of the data source object_id for the given device_id for
+	 *         the case.
+	 *
+	 * @throws TskCoreException if there is a problem getting the data source
+	 *                          obj ids.
+	 */
+	public List<Long> getDataSourceObjIds(String deviceId) throws TskCoreException {
```

Comment:
I don't think this needs to be a public API. I believe the only client is the CommunicationsManager, which is in the same package.